examples/2d/elixir_eulermulti_ec.jl

```diff
@@ -4,7 +4,8 @@ using Trixi
 
 ###############################################################################
 # semidiscretization of the compressible Euler multicomponent equations
-equations = CompressibleEulerMulticomponentEquations2D()
+equations = CompressibleEulerMulticomponentEquations2D(gamma        = (1.4),
+                                                       gas_constant = (0.4))
```

Comment:
```suggestion
equations = CompressibleEulerMulticomponentEquations2D(gamma        = (1.4,),
                                                       gas_constant = (0.4,))
```
Might be better to be explicit about passing a tuple here.

Reply:
done in 42428a4